app/api/routes/news.py

```
@@ -42,3 +42,25 @@ async def get_news(
             status_code=status.HTTP_503_SERVICE_UNAVAILABLE,
             detail=f"Error fetching news: {str(e)}",
         )
+
+
+@router.get("/{artilce_id}", response_model=SingleNewsArticleResponse)
+async def get_news_by_id(artilce_id: str):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)